docs/how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/set-up-clusters-from-hosted-kubernetes-providers/eks.md

```diff
@@ -88,12 +88,13 @@ This [tutorial](https://aws.amazon.com/blogs/opensource/managing-eks-clusters-ra
 
 ## Minimum EKS Permissions
 
-These are the minimum set of permissions necessary to access the full functionality of Rancher's EKS driver. You'll need additional permissions for Rancher to provision the `Service Role` and `VPC` resources. If you create these resources **before** you create the cluster, they'll be available when you configure the cluster.
+These are the minimum set of permissions necessary to access the full functionality of Rancher's EKS driver. You'll need additional permissions for Rancher to provision the `Service Role` and `VPC` resources. If you create these resources **before** you create the cluster, they'll be available when you configure the cluster. Additionally, starting with EKS v1.23 you are required to use the out-of-tree drivers for EBS backed volumes and you will need specific permissions to enable the add-on.
```

Comment:
```suggestion
These are the minimum set of permissions necessary to access the full functionality of Rancher's EKS driver. You'll need additional permissions for Rancher to provision the `Service Role` and `VPC` resources. If you create these resources **before** you create the cluster, they'll be available when you configure the cluster. 

:::note 
In EKS v1.23 and above, you must use the out-of-tree drivers for EBS backed volumes. You need the EBS CSI Driver add-on permissions to enable the add-on.
:::
```
"EBS CSI Driver add-on" == the specific permission? We should mention the v1.23+ info within the description for the resource in the table.

Reply:
`EBS CSI Driver add-on` refers to the name of the Amazon add-on that must be installed in the EKS cluster to allow interaction with EBS volumes. To install and use this add-on, it is required to add these extra IAM permissions.